tests/generation/SettingsTest.py

```diff
@@ -1,3 +1,4 @@
+import ntpath
```

Comment:
Is there a specific reason we are using `ntpath` instead of `os.path`?

Reply:
i wasn't working earlier with `os.path` but i wasn't using `normpath` before, i will surely look into that